packages/invoices/src/InboundShipment/DetailView/GeneralTab.tsx

```diff
@@ -35,48 +43,188 @@ const Expand: FC<{ rowData: InboundShipmentItem }> = ({ rowData }) => {
   );
 };
 
-export const GeneralTabComponent: FC<GeneralTabProps<InboundShipmentItem>> = ({
-  onRowClick,
-}) => {
+export const GeneralTabComponent: FC<
+  GeneralTabProps<InboundShipmentItem | InvoiceLine>
+> = ({ onRowClick }) => {
   const { pagination } = usePagination();
   const t = useTranslation('common');
 
-  const { data, sortBy, onSort } = useInboundItems();
-  const activeRows = data?.filter(({ isDeleted }) => !isDeleted) ?? [];
+  const lines = useInboundLines();
+  const { data: items } = useInboundItems();
```

Comment:
the `lines` and `items` are 'picked' from the `inbound` query and memoised within their hooks.. so just switch between them based on `isGrouped`